eras/shelley/impl/src/Cardano/Ledger/Shelley/LedgerState/IncrementalStake.hs

```diff
@@ -220,19 +241,19 @@ incrementalStakeDistr pp (IStake credStake ptrStake) ds ps =
       if ignorePtrs
         then activeCreds
         else -- Resolve inserts and delets which were indexed by ptrs, by looking them up in the ptrsMap
-        -- and combining the result of the lookup with teh ordinary stake, keeping only the active credentials
+        -- and combining the result of the lookup with the ordinary stake, keeping only the active credentials
           Map.foldlWithKey' addResolvedPointer activeCreds ptrStake
     step2 = aggregateActiveStake triplesMap step1
     addResolvedPointer ans ptr coin =
       case Map.lookup ptr ptrsMap of -- map of ptrs to credentials
-        Just cred | VMap.member cred delegs_ -> Map.insertWith (<>) cred coin ans
+        Just cred | VMap.member cred delegs_ -> Map.insertWith (<>) cred (compactCoinOrError coin) ans
```

Comment:
Now that you made `incrementalAggregateUtxoCoinByCredential` avoid negative coins, could we switch IStake to use `CompactForm Coin` too? At least the `ptrMap` part of that type. This would avoid this partial call here.

Reply:
done